service/controller/key/common.go

```diff
@@ -188,6 +196,11 @@ func IsV19Release(releaseVersion *semver.Version) bool {
 	return releaseVersion.Major >= v19.Major
 }
 
+func IsMinimumFlatcarVersion(flatcarVersion *semver.Version) bool {
+	minRelease, _ := semver.New("3689.0.0")
```

Comment:
if we hardcode this, it means we have to remember to bump it every time we release aws operator in the future.
How about taking the value from the release CR and only allowing anything newer than that?

Reply:
That make sense, I changed it 👍🏻